gaiku-common/src/data/chunk.rs

```diff
@@ -59,7 +59,7 @@ impl Chunk {
     pub fn get_color(&self, x: usize, y: usize, z: usize) -> Option<Vector4<u8>> {
```

Comment:
We can also get rid of the optional here, because we initialize the colors array we know that we will get always a value.

Reply:
This might will require some changes elsewhere too.

Reply:
If we specify an X greater or less than bounds it will return a None I think.

Reply:
Perhaps this should all be reworked to to a hashmap instead of fixed sized arrays.

Reply:
I'm not sure if hashmaps are good in terms of performance/optimization, I need to check that out but I think the hashmap can have spread their content in memory while arrays are continuous (I can be wrong here), but also we need to check how does a change like this affects the performance too.